controllers/kogitoserverlessworkflow_controller.go

```diff
@@ -29,9 +29,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
+	"github.com/kiegroup/kogito-serverless-operator/api"
```

Comment:
operatorapi ?

Reply:
The operatorapi is this one:

```
operatorapi "github.com/kiegroup/kogito-serverless-operator/api/v1alpha08"
```

It's the versioned one where we can change only the import and keep the alias.

In this case, `api` is the better name.